File: packages/rust/tests/fixtures/one_function/typescript/clean/commented.ts
/** Return the value unchanged. */
export function described(value: number): number {
  // The identity is the whole contract.
  //
  // A blank line follows this comment block.

  return value;
}

export function compute(value: number): number {
  const scaled = value * 2;
  return scaled + 1;
}

File: packages/rust/tests/fixtures/one_function/rust/clean/commented.rs
/// Return the value unchanged.
pub fn described(value: u8) -> u8 {
    // The identity is the whole contract.
    //
    // A blank line follows this comment block.

    value
}

pub fn compute(value: u8) -> u8 {
    let scaled = value * 2;
    scaled + 1
}
